Runtime/Scripts/RGBotConfigs/RGState.cs

```diff
@@ -93,7 +93,20 @@ public static IRGStateEntity GenerateCoreStateForRGEntity(RGEntity rgEntity)
             var theTransform = rgEntity.transform;
             
             state["id"] = theTransform.GetInstanceID();
-            state["type"] = rgEntity.objectType;
+            // default to the gameObject name without uniqueness numbers
+            var otName = rgEntity.objectType.Trim();
+            if (string.IsNullOrEmpty(otName))
+            {
+                var goName = rgEntity.gameObject.name;
+                var index = goName.IndexOf('(');
+                if (index > 0)
+                {
+                    goName = goName.Substring(0, index);
```

Comment:
(Non-blocking) C# has a slice syntax you can use here, and I believe Unity's version of C# supports it. Purely an aesthetic choice though, it compiles to a call to Substring:

```csharp
goName = goName[0..index]
```